app/helpers/blacklight/url_helper_behavior.rb

```diff
@@ -72,7 +72,7 @@ def session_tracking_path document, params = {}
     if respond_to?(controller_tracking_method)
       send(controller_tracking_method, params.merge(id: document))
     else
-      blacklight.track_search_context_path(params.merge(id: document))
+      main_app.track_catalog_path(params.merge(id: document))
```

Comment:
Should there be a log message here warning that falling back to an unsafe default?

Reply:
Yeah, it would be nice if these track path worked with more than just catalog... we usually have to add these after the fact in order to make our build not fail (i.e. track for articles vs catalog etc.)

Reply:
I think that logging a message about the controller_tracking_method not being defined and falling back on the unsafe default, perhaps with a pointer to some docs, would be a good addition here.